FEM/rf_pcs.cpp

```diff
@@ -26,6 +26,9 @@
 /*--------------------- MPI Parallel  -------------------*/
 #if defined(USE_MPI) || defined(USE_MPI_PARPROC) || defined(USE_MPI_REGSOIL)
 #include <mpi.h>
+#ifdef OGS_FEM_IPQC
```

Comment:
sorry after #35, we don't need this #ifdef anymore. SplitMPI_Communicator.h is always used if USE_MPI